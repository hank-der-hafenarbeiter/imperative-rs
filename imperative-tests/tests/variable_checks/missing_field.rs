use imperative_rs::InstructionSet;

#[derive(InstructionSet)]
enum Is {
    #[opcode = "0xfvwf"]
    A{v:u8},
}

fn main() {}
