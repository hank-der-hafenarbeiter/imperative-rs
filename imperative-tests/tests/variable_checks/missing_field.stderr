error: Opcode contains w which is neither a valid digit nor a variable name.
 --> $DIR/missing_field.rs:5:16
  |
5 |     #[opcode = "0xfvwf"]
  |                ^^^^^^^^
